src/components/Accordion/__snapshots__/Accordion.spec.jsx.snap

```diff
@@ -1,102 +1,118 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
It's so they can use code search to see how many snapshots people are writing :trollface: